notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -55,10 +55,10 @@
    "source": [
```

Comment:
<p>Compress model weights to INT8 precision using&nbsp;<a href="https://github.com/openvinotoolkit/nncf" target="_blank">NNCF</a> =&gt; Compress model weights to INT8 <strong>or INT4</strong> precision using&nbsp;<a href="https://github.com/openvinotoolkit/nncf" target="_blank">NNCF</a></p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1428/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done</p>

Reply:
<p>again, I would use the notion of "8-bit or 4-bit data types" to make it more generic.</p>

Reply:
<p>Changed</p>